### PR TITLE
Update libxml2 and use xz package - 3.18.x

### PR DESCRIPTION
**Pull request by Lex-2008**
cherry-picked from https://github.com/cfengine/buildscripts/pull/1143

Ticket: ENT-9539

### CODE REVIEW

**Comment by Lex-2008** [comment]
@cf-bottom build exotics here also please

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/8828/)

(with exotics)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/8828/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-8828/

File: deps-packaging/libxml2/cfbuild-libxml2.spec [hunk 1]
```diff
@@ -16,6 +16,9 @@ AutoReqProv: no
 
 %prep
 mkdir -p %{_builddir}
+export PATH=/opt/freeware/bin:$PATH # to use newer version of tar on aix platform
```
Comment (vpodzime) [inline deps-packaging/libxml2/cfbuild-libxml2.spec:19]: Would be nice to have this AIX-specific.

**Review by vpodzime — APPROVED** [review]
Looks good to me.

**Pull request merged**